code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -72,6 +72,12 @@
 		to_chat(src, "<span class='warning'><i>I can't latch onto another slime...</i></span>")
 		return FALSE
 
+	if(isIPC)
```

Comment:
For whatever reason, this is throwing an error. Apparently `isIPC` isn't defined for this instance